scripts/local/.env

```diff
@@ -0,0 +1,52 @@
+# -----------------------------------------------------------------------------
+# Required Environment Variables for Local setup
```

Comment:
nit: Required environment variables for local setup

Reply:
done